ArduPlane/quadplane.cpp

```diff
@@ -333,7 +342,7 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
     // @Bitmask: 0:Level Transition,1:Allow FW Takeoff,2:Allow FW Land,3:Vtol Takeoff Frame,4:Use FW Approach,5:Use QRTL,6:Use Governor,7:Force Qassist,8:Mtrs_Only_Qassist,9:Airmode_On_Arm,10:Disarmed Yaw Tilt,11:Delay Spoolup,12:disable Qassist based on synthetic airspeed,13:Disable Ground Effect Compensation,14:Ignore forward flight angle limits in Qmodes
     AP_GROUPINFO("OPTIONS", 58, QuadPlane, options, 0),
 
-    AP_SUBGROUPEXTENSION("",59, QuadPlane, var_info2),
+    AP_SUBGROUPEXTENSION("",60, QuadPlane, var_info2),  // 59 is used above
```

Comment:
I think this has to stay at 59, and sneak the new one in at 60.

Reply:
compiles fine either way...my understanding (could be wrong) and that the second group all resides in that  AP_SUBGROUPEXTENSION at the first groups #59 param slot in the tree...and that 63 params are allowed in that first group..so it seems like that param number should move down and additions to the first group be above it....
..whether you move the slot or not does not seem to impact the compile...maybe something else

and why add in the first group and not the second? to allow it to be put next to its associated param....although GCS sort alphabetically, the parm list in the wiki is generated in order.

Reply:
I think you will loose all the saved param for the second group if you move it. To test you would have to change them on master and then load this branch and compare.

Reply:
since it compiles either way, I will not move it....

Reply:
done